server/game/cards/01_SOR/ImperialInterceptor.ts

```diff
@@ -0,0 +1,29 @@
+import AbilityHelper from '../../AbilityHelper';
+import { NonLeaderUnitCard } from '../../core/card/NonLeaderUnitCard';
+import { Location, RelativePlayer, WildcardCardType } from '../../core/Constants';
+
+export default class ImperialInterceptor extends NonLeaderUnitCard {
+    protected override getImplementationId() {
+        return {
+            id: '9002021213',
+            internalName: 'imperial-interceptor'
+        };
+    }
+
+    public override setupCardAbilities() {
+        this.addWhenPlayedAbility({
+            title: 'You may deal 3 damage to a space unit',
```

Comment:
Hey sorry, missed this on the first pass - another style thing is we don't use "You" language for the ability titles like this. So I think in this case it would just be "Deal 3 damage to a space unit". We also don't include "may" since that is implied by the presence of the "Pass ability button"

Reply:
No worries all fixed.